kubernetes/overlays/delegation-process/qa/configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{SERVICE_NAME}}
+  namespace: {{NAMESPACE}}
+data:
+  EVENTSTORE_DB_SCHEMA: "{{NAMESPACE}}_delegation"
+  ACCEPTED_AUDIENCES: "{{UI_JWT_AUDIENCE}},{{M2M_JWT_AUDIENCE}},{{INTERNAL_JWT_AUDIENCE}}"
+  S3_BUCKET: "{{CATALOG_PROCESS_STORAGE_CONTAINER}}"
```

Comment:
Here I put `CATALOG_PROCESS_STORAGE_CONTAINER` when I created the overlay in DEV (since they use the same bucket), but it's probably better to put `DELEGATION_CONTRACTS_CONTAINER`